scripts/python/amend_user_repository_access.py

```diff
@@ -528,36 +528,6 @@ def fetch_teams(gql_client: Client) -> list:
     return teams_list
 
 
-def remove_user_from_repository(github_service: GithubService, user_name, repository_name):
```

Comment:
A few things to note with the refactors:

- Removed timeout - rate limiting functionality will be handled centrally after all functions have been migrated.
- Removed exception handling - prevent us from failing silently